src/wallet/CMakeLists.txt

```diff
@@ -148,7 +124,6 @@ if (BUILD_GUI_DEPS)
     endif()
     install(TARGETS wallet_merged
         ARCHIVE DESTINATION ${lib_folder})
-
-    install(FILES ${wallet_api_headers}
-        DESTINATION include/wallet)
 endif()
+
+add_subdirectory(api)
```

Comment:
Don't we want to have `if (BUILD_GUI_DEPS)` here? Otherwise `wallet_api` always gets rebuilt every time `wallet2.h` gets changed.

Reply:
Good point, I'd been testing with making simplewallet, but maybe this gets rebuilt with simple make, testing now.